test/Microsoft.AspNetCore.Server.IISIntegration.Tests/IISMiddlewareTests.cs

```diff
@@ -150,15 +156,62 @@ public void PathBaseHiddenFromServer()
                 .UseSetting("PORT", "12345")
                 .UseSetting("APPL_PATH", "/")
                 .UseIISIntegration()
+                .ConfigureServices(services => services.AddAuthentication())
```

Comment:
Well we can't even call GetScheme without AddAuthentication as the AuthSchemeProvider, we already have tests that verify that the server can start without AddAuthentication.